iosApp/iosApp/Pages/StopDetails/StopDetailsFilteredDepartureDetails.swift

```diff
@@ -60,10 +62,12 @@ struct StopDetailsFilteredDepartureDetails: View {
                             }
                         )
                         .fixedSize(horizontal: false, vertical: true)
-                        .padding(.horizontal, 16)
-                        .padding(.top, 16)
+                        .padding([.horizontal, .top], 16)
+                        .padding(.bottom, 6)
+                        .dynamicTypeSize(...DynamicTypeSize.accessibility1)
 
                         departureTiles(patternsByStop, view)
+                            .dynamicTypeSize(...DynamicTypeSize.accessibility3)
```

Comment:
We should probably be using it in more places on this page and elsewhere, it would be good to do a full UI pass at some point to ensure that all the different text sizes work everywhere and constrain or make the UI more flexible as needed, the GL route pills are especially painful and have been funky for a long time.

Reply:
Love that idea - could you make a ticket for it?

Reply:
[Done](https://app.asana.com/0/1205425564113216/1208939101023568)!